bot/exts/core/help.py

```diff
@@ -313,6 +313,8 @@ async def _add_command_signature(self, paginator: LinePaginator) -> None:
         prefix = constants.Client.prefix
 
         signature = self._get_command_params(self.query)
+        paginator.add_line(f"**```\n{prefix}{signature}\n```**")
+
         parent = self.query.full_parent_name + " " if self.query.parent else ""
         paginator.add_line(f"**```\n{prefix}{parent}{signature}\n```**")
```

Comment:
Missed this line being removed in my patch, currently the top line is duplicated
![image](https://user-images.githubusercontent.com/22353562/185453470-be35be1c-9ebc-48d1-b375-079a9fde13de.png)

```suggestion
```

Reply:
Also with that we can enable the help command in DMs

![image](https://user-images.githubusercontent.com/22353562/185455557-d303d8d4-0f68-4c5e-b950-c0cedf5e8b4b.png)

Reply:
[`1de56aa` (#1049)](https://github.com/python-discord/sir-lancebot/pull/1049/commits/1de56aa67f0d5480b2016736bf6dcd6dd4bbff72) and [`050ec2b` (#1049)](https://github.com/python-discord/sir-lancebot/pull/1049/commits/050ec2bf36bc70415f07f4f0aaf3e911e4af118f)